utils/mariadb/Makefile

```diff
@@ -0,0 +1,215 @@
+#
+# Copyright (C) 2017 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/uclibc++.mk
+
+PKG_NAME:=mariadb
+PKG_VERSION:=5.5.54
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-galera-$(PKG_VERSION).tar.gz
```

Comment:
galera is the correct way to do it because non galera source will not have option for clusters